tools/android_lint/project.xml

```diff
@@ -1,81 +1,81 @@
 <!-- THIS FILE IS GENERATED. PLEASE USE THE INCLUDED DART PROGRAM  WHICH -->
 <!-- WILL AUTOMATICALLY FIND ALL .java FILES AND INCLUDE THEM HERE       -->
 <project>
-  <sdk dir="../../../third_party/android_tools/sdk" />
+  <sdk dir="../third_party/android_tools/sdk" />
```

Comment:
I'm assuming this changed because you ran it from a different directory than I did, but it surprises me that this works.  Shouldn't this be relative to the dir it's in, rather than to the dir that it's run from?

Reply:
I re-ran the tool in this directory to avoid the file changes. I'm not sure why it depends on run location.